cf/test/test_Data.py

```diff
@@ -913,9 +901,7 @@ def test_Data_flatten(self):
 
     @unittest.skipIf(TEST_DASKIFIED_ONLY, "no attribute 'partitions'")
```

Comment:
@davidhassell I suspect this test can be removed, as it is LAMA-centered, but I am not 100% sure?

Reply:
Yes, this can go.

Reply:
Thanks for clarifying, I will remove it in a stand-alone commit.